### DISCUSSION TITLE
Conical Slicing - No Planar 3D Printing.

### DISCUSSION THREAD

**Discussion by carlymx** [supermerill/SuperSlicer#3431] [General]
![Conick Slicing](https://xyzdims.com/wp-content/uploads/2021/02/overhang-anim2.gif)

ENG Translator:
Hello, good afternoon.

For some time we have been hearing about the possibility of using our old printers to print parts in a "non-Planar" way. There is no doubt that it would be a significant advance in filleting technology and 3D printing itself.

For the crack programmers of this magnificent program called SuperSlicer :) . Are there plans to implement this technology?

Info about Conical Slicing:
The info: https://xyzdims.com/2021/02/26/3d-printing-conic-slicing-for-rotating-tilted-nozzle-rtn/
A video: https://www.youtube.com/watch?v=1i-1TEdByZY
A Guide for SuperSlicer: https://www.cnckitchen.com/blog/guide-how-to-use-conical-slicing

(Original ESP)
Hola buenas tardes.

Desde hace algún tiempo se viene escuchando la posibilidad de utilizar nuestras viejas impresoras para imprimir piezas de forma "no Planar" No cave duda que sería un avance significativo en la tecnología de fileteado y de la propia impresión 3D.

Para los cracks programadores de este magnífico programa llamado SuperSlicer :) .  ¿Hay planes para implementar esta tecnología?.

Info sobre Conical Slicing:
The info: https://xyzdims.com/2021/02/26/3d-printing-conic-slicing-for-rotating-tilted-nozzle-rtn/
A video: https://www.youtube.com/watch?v=1i-1TEdByZY
A Guide for SuperSlicer: https://www.cnckitchen.com/blog/guide-how-to-use-conical-slicing

**Comment by vovodroid** [comment]
There is printing with modified Slick3r version: https://youtu.be/gmePlcU0TRw
